src/ServiceBus/ServiceBus/Cmdlets/NetworkRuleSet/GetAzureServiceBusNetworkRuleSet.cs

```diff
@@ -0,0 +1,90 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ServiceBus.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.ServiceBus.Commands.NetworkruleSet
+{    
+    /// <summary>
+    /// 'Get-AzureRmEventHub' Cmdlet gives the details of a / List of EventHub(s)
+    /// <para> If EventHub name provided, a single EventHub detials will be returned</para>
+    /// <para> If EventHub name not provided, list of EventHub will be returned</para>
+    /// </summary>
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ServiceBusNetworkRuleSet", DefaultParameterSetName = NetwrokruleSetPropertiesParameterSet), OutputType(typeof(PSNetworkRuleSetAttributes))]
+    public class GetAzureServiceBusNetworkRuleSet : AzureServiceBusCmdletBase
+    {
+        [Parameter(Mandatory = true, ParameterSetName = NetwrokruleSetPropertiesParameterSet, Position = 0, HelpMessage = "Resource Group Name")]
+        [Parameter(Mandatory = false, ParameterSetName = NetwrokruleSetNamespacePropertiesParameterSet, Position = 0, HelpMessage = "Resource Group Name")]
```

Comment:
@v-Ajnava optional parameters shouldn't have a `Position` property

This comment applies to all parameters in this PR that are optional and have the `Position` property